xml/gpu_passthru.xml

```diff
@@ -0,0 +1,503 @@
+   <command>virt-install</command> to install the guest VM. Find more details
+   in <xref linkend="cha-kvm-inst"/>.
+  </para>
+
+  <sect2 xml:id="gpu-passthru-guest-general">
+   <title>Requirements for the guest configuration</title>
+   <para>
+    During the guest VM installation, select <guimenu>Customize configuration
+    before install</guimenu> and configure the following devices:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Use Q35 chipset if possible.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Install the guest VM by using the UEFI firmware.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Add the following emulated devices:
+     </para>
+     <para>
+      Graphic: Spice or VNC
+     </para>
+     <para>
+      Device: qxl, VGA, or Virtio
+     </para>
+     <para>
+      Find more information in <xref linkend="sec-libvirt-config-video"/>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Add the host PCI device (<literal>03:00.0</literal> in our example) to
+      the guest. Find more information in
+      <xref linkend="sec-libvirt-config-pci"/>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      For best performance, we recommend using virtio drivers for the network
+      card and storage.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-guest-driver">
+   <title>Install the graphic card driver</title>
+   <sect3 xml:id="gpu-passthru-guest-driver-linux-rpm">
+    <title>Linux guest</title>
+    <procedure>
+     <title>RPM-based distributions</title>
+     <step>
+      <para>
+       Download the driver RPM package from
+       <link xlink:href="http://www.nvidia.com/download/driverResults.aspx/131159/en-us"/>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Install the downloaded RPM package:
+      </para>
+<screen>&prompt.sudo;rpm -i nvidia-diag-driver-local-repo-sles123-390.30-1.0-1.x86_64.rpm</screen>
+     </step>
+     <step>
+      <para>
+       Refresh repositories and install <package>cuda-drivers</package>. This
+       step will be different for non-&suse; distributions:
+      </para>
+<screen>&prompt.sudo;zypper refresh &amp;&amp; zypper install cuda-drivers</screen>
+     </step>
+     <step>
+      <para>
+       Reboot the guest VM:
+      </para>
+<screen>&prompt.sudo;shutdown -r now</screen>
+     </step>
+    </procedure>
+    <procedure>
+     <title>Generic installer</title>
+     <step>
+      <para>
+       Because the installer needs to compile the &nvidia; driver modules,
+       install <package>gcc-c++</package> and <package>kernel-devel</package>
+       packages.
+      </para>
+     </step>
+     <step>
+      <para>
+       Disable Secure Boot on the guest, because the &nvidia; driver modules are not
+       signed. On &suse; distributions, you can use &yast; &grub; module to
+       disable Secure Boot. Find more information in
```

Comment:
```suggestion
       Disable Secure Boot on the guest, because &nvidia;'s driver modules are
       unsigned. On &suse; distributions, you can use the &yast; &grub; module
       to disable Secure Boot. Find more information in
```